spec/index.bs

```diff
@@ -7964,12 +7964,24 @@ attachments used by this encoder.
 
             **Arguments:**
             <pre class=argumentdef for="GPURenderPassEncoder/executeBundles(bundles)">
-                bundles: List of render bundles to execute.
+                |bundles|: List of render bundles to execute.
             </pre>
 
             **Returns:** {{undefined}}
 
-            Issue: Describe {{GPURenderPassEncoder/executeBundles()}} algorithm steps.
+            Issue the following steps on the [=Device timeline=] of |this|:
+            <div class=device-timeline>
+                1. If any of the following conditions are unsatisfied, generate a validation
+                    error and stop.
+                    <div class=validusage>
+                        - For each |bundle| in |bundles|:
+                            - |bundle| must be [$valid to use with$] |this|.
+                            - |this|.{{GPURenderEncoderBase/[[layout]]}} must equal |bundle|.{{GPURenderBundle/[[layout]]}}.
+                            - |this|.{{GPURenderEncoderBase/[[depthReadOnly]]}} must equal to |bundle|.{{GPURenderBundle/[[depthReadOnly]]}}.
```

Comment:
This is a little stricter than SetPipeline. But It is easy and make senses:
If there is no render bundle and we call RenderPassEncoder.SetPipeline(), it requires that if pipeline.writesDepth is true, RenderPassAttachment.depthReadOnly must be false. 
If there is render bundle between render pipeline and render pass. Render bundle acts like a bridge between render pipeline and render pass. For RenderBundleEncoder.SetPipeline(), it requires that if pipeline.writesDepth is true, RenderBundleEncoder.depthReadOnly must be false. Then for RenderPassEncoder.executeBundles, we can require that RenderBundle.depthReadOnly equals to RenderPassEncoder.depthReadOnly because: 1) for most cases, depthReadOnly (in bundle or pass) is default to false, so it doesn't matter if pipeline.writesDepth is true or false. In this case, bundle.depthReadOnly == pass.depthReadOnly == false. 2) if we do want to use readonly depth attachment and set depthReadOnly to true, we should set both of bundle.depthReadOnly and pass.depthReadOnly to be true, and we will use depth attachment as a sampled texture in pipeline, the `Resource usage validation` also require that both depthReadOnly to be true in order to ensure the usage is a readonly one: ATTACHMENT_READ. 3) the only tricky one is that if pipeline.writesDepth is false, RenderBundle.depthReadOnly and PassEncoder.depthReadOnly can be different, `and we DO NOT set a sampled texture for depth attachment`. In this case, this validation is stricter than SetPipeline's validation rule. But this situation is not very useful: why do we need to set depthReadOnly to true if we don't sample from it. If we don't want to write into depth attachment, then don't enable depth write, that's enough. It is not necessary to set depthReadOnly to true.